.github/workflows/test.yml

```diff
@@ -31,10 +31,23 @@ jobs:
           from json import dumps
 
           matrix = [
-            {"kernel": "LATEST", "runs_on": ["ubuntu-latest", "self-hosted"], "arch": "x86_64", "toolchain": "gcc"},
-            {"kernel": "LATEST", "runs_on": ["ubuntu-latest", "self-hosted"], "arch": "x86_64", "toolchain": "${{ needs.llvm-toolchain.outputs.llvm }}"},
-            {"kernel": "LATEST", "runs_on": ["z15", "self-hosted"], "arch": "s390x", "toolchain": "gcc"},
+            {"kernel": "LATEST", "runs_on": ["ubuntu-latest"], "arch": "x86_64", "toolchain": "gcc"},
+            {"kernel": "LATEST", "runs_on": ["ubuntu-latest"], "arch": "x86_64", "toolchain": "${{ needs.llvm-toolchain.outputs.llvm }}"},
+            {"kernel": "LATEST", "runs_on": ["z15"], "arch": "s390x", "toolchain": "gcc"},
           ]
+
+          if "${{ github.repository_owner }}" == "kernel-patches":
```

Comment:
one thing I thought about after approving.

what do you think about keeping self-hosted in the array and then have one loop filtering for z15 / self-hosted and removing them.

it will simplify the logic a bit

Reply:
Sure, we can do that.